samples/global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "3.1.302"
+  }
+}
```

Comment:
This file may not be needed, but when opening the Samples.sln in VS2019, for me it doesn't discover the `global.json` at the root of the repo and refuses to load the `.csproj` until I add this file here.